src/config/dictionary/index.js

```diff
@@ -0,0 +1,3 @@
+export default {
+    
+}
```

Comment:
what is this dictionary ?

Reply:
I will move [this part](https://github.com/nemfoundation/nem2-explorer/blob/efdc6369850bfb54483e163aec262d5202777106/src/store/ui.js#L25) here for different languages

Reply:
may we can call it `i18n`

Reply:
Yes, that's better, thanks